### ISSUE TITLE
Progress bars don't convey status to users of assistive technology

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#3718]
There are a few places in the Pressbooks interface where we have dynamic progress bars that inform sighted users of progress towards a lengthy task (imports, exports, cloning routine). An a11y specialist at Wilfrid Laurier University has reported that there is no indication of progress announced for users of JAWS (screenreader software):  https://pressbooks.zendesk.com/agent/tickets/20515. There should be.

To reproduce:
1. As a sighted user importing a Pressbook.xml file, select the Import Selection button and observe the import progress bar.
    
![image001](https://github.com/pressbooks/pressbooks/assets/13485451/8849162f-b3db-481c-bcc4-2666c43ea23d)

(This screenshot shows meter depicting current level of progress in uploading a file)
2. Perform this same task using the JAWS screenreader software. Upon importing a Pressbook.xml file and selecting the Import Selection button, you hear no status updates. You wonder whether anything is happening. 

Problem:
The visual interfaces conveys progress, while the aural interface has no indicators of progress.
 
Technical solution:
Use a progress bar with relevant ARIA roles, states, and properties: https://w3c.github.io/aria/#progressbar + https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/progressbar_role#associated_wai-aria_roles_states_and_properties. For further reading, consult https://www.w3.org/TR/wai-aria-1.1/#progressbar and https://pressbooks.library.torontomu.ca/wafd/chapter/progress-bars/

See https://pressbooks.org/blog/2018/11/12/refactoring-slow-forms-using-php-generators-and-event-streams/, https://github.com/pressbooks/pressbooks/issues/1282, and https://github.com/pressbooks/pressbooks/pull/1589 for implementation. It looks to me from the PR that aria updates were a part of the initial implementation, but a user has reported not hearing updates when performing the import action.

**Comment by ho-man-chan** [comment]
@arzola will take a look.

**Comment by tw77** [comment]
Progress bars on dev are announcing progress in percentages (tested exports, imports, clones, cover generation with VoiceOver). Ready for release
